src/main.c

```diff
@@ -351,13 +356,75 @@ key_binding_func_key (MBWindowManager   *wm,
   hd_dbus_send_event (s);
 }
 
+/* Fill in the array with input devices type is touchscreen */
+static void
+enumerate_input_devices (Display *dpy)
+{
+  static Atom atom_touchscreen = None;
+
+  if (atom_touchscreen == None)
+    atom_touchscreen = XInternAtom (dpy, XI_TOUCHSCREEN, True);
+
+  if (atom_touchscreen != None)
+    {
+      XDeviceInfo *devinfo;
+      int i, ndev, nclass = 0;
+      XEventClass eclass[64];
+
+      /* get XInput DeviceMotion events */
+      devinfo = XListInputDevices (dpy, &ndev);
+
+      if (ndev > G_N_ELEMENTS(eclass))
+        ndev = G_N_ELEMENTS(eclass);
+
+      if (ndev > G_N_ELEMENTS(xdevice_is_ts))
+        ndev = G_N_ELEMENTS(xdevice_is_ts);
+
+      for (i = 0; i < ndev; i++)
+        {
+          XDeviceInfo info = devinfo[i];
+
+          if (info.use == IsXExtensionPointer)
+            {
+              XDevice *dev = XOpenDevice (dpy, info.id);
+
+              /* bizzaro macro, to SET arg 2 and 3 */
```

Comment:
what is bizzaro?

Reply:
This is the original author comment, I'll leave it here. See https://github.com/Cordia/hildon-desktop/commit/df2dd4a9514504285f04f4a6a26e43440b7346f5#diff-803c5170888b8642f2a97e5e9423d399R843

Reply:
Sure, if you prefer to leave something that you don't understand. Was it a typo?

Reply:
Bizzaro is slang, it's not a typo. It's like "bizarre"

Reply:
anyway, I am removing it in the new version of the PR